### ISSUE TITLE
Error with Ansible 2.4

### ISSUE THREAD

**Issue by gtirloni** [philpep/testinfra#249]
**Versions:**

- testinfra 1.7.0
- ansible 2.4.0
- molecule 1.25

**Error:**

```
____________________ ERROR collecting tests/test_default.py ____________________
tests/test_default.py:5: in <module>
    '.molecule/ansible_inventory').get_hosts('all')
venv/lib/python2.7/site-packages/testinfra/utils/ansible_runner.py:150: in __init__
    self.variable_manager = ansible.vars.VariableManager()
E   AttributeError: 'module' object has no attribute 'VariableManager'
```

**test_default.py:**

```
import testinfra.utils.ansible_runner

testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
    '.molecule/ansible_inventory').get_hosts('all')
```

**Comment by gtirloni** [comment]
@philpep thank you!

**Comment by philpep** [comment]
Fixed and released in 1.7.1
